Directory.Packages.props

```diff
@@ -120,7 +120,7 @@
     <PackageVersion Include="xunit" Version="$(_XunitPackageVersion)" />
     <PackageVersion Include="Xunit.Combinatorial" Version="1.5.25" />
     <PackageVersion Include="xunit.extensibility.execution" Version="$(_XunitPackageVersion)" />
-    <PackageVersion Include="xunit.runner.utility" Version="2.4.1" />
+    <PackageVersion Include="xunit.runner.utility" Version="$(_XunitPackageVersion)" />
```

Comment:
I haven't looked, but is `_XunitPackageVersion` >= `2.4.1`?

Reply:
Currently 2.6.3 by the look of things